Runtime/Game/Resources/LootLockerConfig.cs

```diff
@@ -105,6 +110,11 @@ static void ListRequestProgress()
                         break;
                     }
                 }
+
+                if (string.IsNullOrEmpty(LootLockerConfig.current.sdk_version))
+                {
+                    LootLockerConfig.current.sdk_version = LootLockerJson.DeserializeObject<LLPackageDescription>(File.ReadAllText("Assets/LootLockerSDK/package.json")).version;
```

Comment:
This will break if LootLocker was added somewhere else in your project. A common folder structure in Unity Projects is to put all external content in something like /ThirdParty or /External or similar.
I suggest to add a check to see if this file exists before trying to read it.
And even more optimal would be to search for the file in the project if tit couldn't be found,

Reply:
Fixed